### ISSUE TITLE
Simplest Policy Gradient(SPG)

### ISSUE THREAD

**Issue by TypingCat** [TypingCat/wheel#14]
This is the first implementation of policy gradient. I refer to [SPG](https://spinningup.openai.com/en/latest/spinningup/rl_intro3.html) of the OpenAI Spinning Up. It only has 3 steps:

1. Build a policy network
2. Set a loss function
3. Learning policy from experiences

**Comment by TypingCat** [comment]
According to the second feature of https://github.com/TypingCat/wheel/issues/27#issuecomment-772199733, it seems that SPG can also use continuous action space. I'd like to know if changing action space domain can improves performance.

**Comment by TypingCat** [comment]
Categorical actor was replaced by Gaussian actor on commit 312958cc51dce60032f9feca1daff4b7610304c6. The agent still falls into the local minima. It usually moves in a circle. No data is accumulated because the scenario is not finished. Decreasing the learning rate and increasing the batch size had no effect.
